app/views/organizations/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :title, "New Organization" %>
+
+<% content_for :content_size, "col-sm-9 col-md-10" %>
+
+<% content_for :content do %>
+  <%= form_for @organization, data: { live_guarded: true } do |f| %>
+    <div>
+      <div class="form-group">
+        <label for="organization[name]">Name</label>
```

Comment:
You are using form helpers for the inputs, why not the labels?
`<%= f.label :name %>`